Rakefile

```diff
@@ -50,4 +52,4 @@ end
 # Default
 task default: ['style', 'spec', 'integration:vagrant']
 task ec2: ['style', 'spec', 'integration:ec2']
-task test: ['style', 'spec']
+task test: %w(style spec)
```

Comment:
Due to the number of threads that could spin up with this one, consider updating the Rakefile to use Kitchen::CLI instead of threads (new format in the custom skel).